arlas-core/src/main/java/io/arlas/server/utils/ElasticTool.java

```diff
@@ -77,6 +80,48 @@ public static boolean checkIndexMappingFields(Client client, String index, Strin
         return true;
     }
 
+    public static boolean checkAliasMappingFields(Client client, String alias, String typeName, String... fields) throws ArlasException {
+        List<String> indeces = ElasticTool.getIndecesName(client, alias, typeName);
+        for (String index : indeces) {
```

Comment:
You may write it in one line: indeces.forEach(index => checkIndexMappingFields(client, index, typeName, fields);)